diff -p -up ./Makefile.orig ./Makefile
--- ./Makefile.orig	2010-03-26 08:31:07.000000000 +0100
+++ ./Makefile	2010-03-26 08:33:01.000000000 +0100
@@ -2,9 +2,9 @@
 
 CC = gcc
 
-CFLAGS = -g -I/usr/X11R6/include `pkg-config --cflags gtk+-2.0` -Wall
-DFLAGS = -L/usr/X11R6/lib `pkg-config --libs gtk+-2.0`  -Wall
-LIBS = -lm
+CFLAGS += -I/usr/X11R6/include `pkg-config --cflags gtk+-2.0` -Wall
+DFLAGS += -L/usr/X11R6/lib -Wall
+LIBS = -lgtk-x11-2.0 -lgdk_pixbuf-2.0 -lglib-2.0 
 
 OBJS = main.o setup_menu.o callback_menu.o help.o options.o \
        display.o read_db.o globals.o read_line.o \

